engine/src/main/java/io/camunda/zeebe/engine/metrics/ProcessEngineMetrics.java

```diff
@@ -87,4 +92,16 @@ private boolean isRootProcessInstance(
       final BpmnElementType elementType, final long parentProcessInstanceKey) {
     return isProcessInstance(elementType) && parentProcessInstanceKey == -1;
   }
+
+  public void increaseSuccessfullyEvaluatedDmnElements(final double amount) {
```

Comment:
:wrench: We could change the parameter type of `amount` to `int`. We don't expect floating-point values of partially evaluated decisions.

Reply:
done